sys/net/network_layer/ng_sixlowpan/frag/rbuf.c

```diff
@@ -83,10 +79,12 @@ void rbuf_add(ng_netif_hdr_t *netif_hdr, ng_sixlowpan_frag_t *frag,
     ptr = entry->ints;
 
     /* dispatches in the first fragment are ignored */
-    if (offset != 0) {
-        switch (((uint8_t *)(entry->pkt->data))[0]) {
+    if (offset == 0) {
+        switch (data[0]) {
             case NG_SIXLOWPAN_UNCOMPRESSED:
-                offset++;
```

Comment:
Not directly related to this PR, but are any more cases foreseen here or wouldn't it be simpler to replace the switch-case by an if statement?